src/mixer/plugins/AlsaMixerPlugin.cxx

```diff
@@ -81,7 +81,7 @@ class AlsaMixer final : public Mixer {
 	snd_mixer_t *handle;
 	snd_mixer_elem_t *elem;
 
-	AlsaMixerMonitor *monitor;
+	std::unique_ptr<AlsaMixerMonitor> monitor;
```

Comment:
This one is pointless overhead, because it's going to be deleted in `Close()`, which is guaranteed to be called if `Open()` has been successful. `~AlsaMixer()` calling `~unique_ptr<AlsaMixerMonitor>` will never see any non-nullptr value, thus the overhead for checking again is pointless.
Clearly, this isn't proper RAII C++ code, but your change doesn't improve this in any way. It just puts some lipstick on...